script/formatting/formatter.py

```diff
@@ -386,3 +413,19 @@ def check_file(file):
         if file.endswith(".ipynb"):
             notebook_file = os.path.join(EVA_NOTEBOOKS_DIR, file)
             check_notebook_format(notebook_file)
+
+    # CODESPELL
+    LOG.info("Codespell")
+    os.system("codespell eva/")
```

Comment:
I feel we shouldn't break the build for spelling errors in code. What do you think?

Reply:
This will break the build. However, aspell is an interactive checker -- so it won't exit -- will just keep waiting. Locally, we will fix things interactively with the `LINTER` mode.